doc/level1.rst

```diff
@@ -1122,6 +1122,12 @@ take in some circumstances.
 If the user selects this option, the MUA MUST clear both
 ``accounts[addr].secret_key`` and ``accounts[addr].public_key``.
 
+After this key destruction has happened, the MUA is in a configuration
+where it could restart a :ref:`search for an Autocrypt Setup
+Message<setup-message-import>`.  For Level 1 MUAs, this is the
+simplest path for synchronizing secret keys between two MUAs that have
+mistakenly both enabled Autocrypt independently.
+
```

Comment:
i think this paragraph should rather go to "Setup Message Import" section and link to the "destory" section here. I am fine with merging it thus because the wording says "this is the simplest" path which indicates it's not the only possible solution without diving deeper (a 1.1 topic i think).